standards/storing-source-code.md

```diff
@@ -21,19 +21,29 @@ New repositories for products and services live in the
 [ministryofjustice organisation](https://github.com/ministryofjustice)
 on GitHub.
 
-You can use your personal GitHub account (but you should [add your MOJ
-email address to your account](https://help.github.com/articles/adding-an-email-address-to-your-github-account/),
-and maybe [use that for notifications](https://help.github.com/articles/managing-notification-emails-for-organizations/)).
-Ask the Digital Service Desk or a member of the webops team to add you
-to the organisation. Our GitHub organisation requires that you use
-two-factor authorisation.
-
 Repositories should be [clearly named]({{ '/standards/naming-things' | relative_url }}),
 and have an [appropriate licence]({{ '/standards/licencing-software-or-code' | relative_url }})
 and enough documentation that someone new can get started with the
 project.
 
-If the repository contains secrets rather than code, make it private
-and restrict the people who have access to it. Discuss with the
-security engineering team whether you should encrypt the contents of
-the repository.
+### GitHub.com accounts
```

Comment:
`GitHub`, not `GitHub.com`; GitHub Enterprise has never been a thing at MOJ, so we don't need the qualifier.

Reply:
This is referring to accounts on a website, so saying it is github.com makes sense? I appreciate if you say "github" most people think of "github.com" but maybe I'm splitting hairs to be explicit

Reply:
I appreciate the change to Okta (probability of coming through and when?) but I think we should always ask people to have MFA on their github.com account anyway?

Reply:
We should definitely ask them to have MFA, yeah.

`GitHub.com` just sounds weird, is all.